### PR TITLE
update json-ld examples

### PR DESCRIPTION
**Pull request by tcole3**
Updated json-ld examples and added text describing these examples;
Also updated abstract and introduction
@BigBlueHat if edits to abstract and introduction don't conflict with your work in progress, please merge.

### CODE REVIEW

**Comment by tcole3** [comment]
Marked as non-substantive for IPR from ash-nazg.

**Comment by BigBlueHat** [comment]
Rawgit: https://rawgit.com/tcole3/web-annotation/77d0d1b6e068e23eedb92c595e86a17932ef6e21/serialization-html-note/index-respec.html

File: serialization-html-note/index-respec.html [hunk 1]
```diff
@@ -436,64 +490,83 @@
                 "created": "2017-01-26T17:30:04.639Z",
                 "creator": {
                     "type": "Person",
-                    "email": "mail-to:t-cole3@illinois.edu",
-                    "name": "Tim Cole"
+                    "email": "mail-to:mara@example.org",
```
Comment (BigBlueHat) [inline serialization-html-note/index-respec.html]: Psuedo protocol is actually `mailto:`

**Comment by BigBlueHat** [comment]
@tcole3 there's some good stuff in here, but some things I'd like to undo are:
 - text in the intro suggesting that one can only annotate "resources found on the Web"
   - the Model (and friends) are about annotating *anything* and a means to express those annotation *on the Web*
 - the inclusion of the Web Annotation Protocol mention is a bit distracting (the Note doesn't reference it agian)
 - I'd not put the media type in the intro...clutter-y

Otherwise, I think these are good changed.

Also, the addition of the use cases with links to places where they might be used is a great addition. Thank you!

**Comment by BigBlueHat** [comment]
@tcole3 changes look good, thanks!

**Pull request merged**